dev-docs/source/AlgorithmProfiler.rst

```diff
@@ -15,26 +15,25 @@ Available for Linux only.
 Mantid build
 ^^^^^^^^^^^^
 
-To build mantid for profiling run ``cmake`` with the additional option ``-DPROFILE_ALGORITHM_LINUX=ON``.
-Built in such a way mantid creates a dump file ``algotimeregister.out`` in the running directory.
-This file contains the time stamps for start and finish of executed algorithms with ~nanocecond
-precision in a very simple text format.
+To build mantid version with profiling functionality enabled run ``cmake`` with the additional option
+``-DPROFILE_ALGORITHM_LINUX=ON``. Built in such a way mantid creates a dump file ``algotimeregister.out``
+in the running directory. This file contains the time stamps for start and finish of executed algorithms with
+~nanocecond precision in a very simple text format.
 
 Analysing tool
 ^^^^^^^^^^^^^^
 
 The project is available here: https://github.com/nvaytet/mantid-profiler. It provides the nice graphical
-tool to interpret the information from dumped file.
+tool to interpret the information contained in the dumped file.
 
 Windows development
 ^^^^^^^^^^^^^^^^^^^
 
 Precise timers are different for Linux and Windows (chrono is not good enough), so we need to treat them
 separately. The suggestion is either to modify files ``Framework/API/inc/MantidAPI/AlgoTimeRegister.h`` and
 ``Framework/API/src/AlgoTimeRegister.cpp`` by including ``#ifdef __WIN32``, or create the specific files with
-the ``AlgoTimeRegister`` class for Windows and then in ``Framework/API/CMakeLists.txt`` edit the chunk that
-describes option PROFILE_ALGORITHM_LINUX combined with OS defined flags. In this case
-``Framework/API/src/AlgorithmExecuteProfile.cpp`` also needs to be modified or substituted.
-``${CMAKE_SYSTEM_NAME} MATCHES "Windows"``. QueryPerformanceCounter supposed to be used on windows,
-instead of clock_gettime.
+the ``AlgoTimeRegister`` class defined for Windows.  In the second case an inclusion of OS specific files should be
+done correctly in ``Framework/API/CMakeLists.txt`  using ``${CMAKE_SYSTEM_NAME} MATCHES "Windows"``,
+``Framework/API/src/AlgorithmExecuteProfile.cpp`` needs to be modified or substituted according to the type of OS.
+QueryPerformanceCounter supposed to be used on windows, instead of clock_gettime.
```

Comment:
QueryPerformanceCounter **should** to be used on windows, instead of clock_gettime.

Reply:
also add code formatting for "QueryPerformanceCounter" and "clock_gettime"?